meson.build

```
@@ -0,0 +1,113 @@
+project(
+    'com.github.parnold-x.nasc',
+    'vala', 'c', 'cpp',
+    version: '0.5.0',
+    default_options: ['cpp_std=c++11']
+)
+
+gettext_name = 'nasc'
+
+# check vala compiler version
+if meson.get_compiler('vala').version() < '0.28.0'
```

Weird that this works. I would expect something like the following to be required:

```suggestion
if meson.get_compiler('vala').version().version_compare('< 0.28.0')
```

---

Alright, I'll try this when I'm at my PC.